src/component/feature/FeatureView/FeatureSettings/FeatureSettings.tsx

```diff
@@ -1,46 +1,55 @@
 import { useState } from 'react';
 import PageContent from '../../../common/PageContent';
 import { useStyles } from './FeatureSettings.styles';
-
 import { List, ListItem } from '@material-ui/core';
 import ConditionallyRender from '../../../common/ConditionallyRender';
-import FeatureSettingsMetadata from './FeatureSettingsMetadata/FeatureSettingsMetadata';
 import FeatureSettingsProject from './FeatureSettingsProject/FeatureSettingsProject';
+import { useParams } from 'react-router-dom';
+import { IFeatureViewParams } from '../../../../interfaces/params';
+import { FeatureSettingsInformation } from './FeatureSettingsInformation/FeatureSettingsInformation';
 
 const METADATA = 'metadata';
 const PROJECT = 'project';
 
-const FeatureSettings = () => {
+export const FeatureSettings = () => {
     const styles = useStyles();
-
+    const { projectId, featureId } = useParams<IFeatureViewParams>();
     const [settings, setSettings] = useState(METADATA);
 
     return (
         <PageContent headerContent="Settings" bodyClass={styles.bodyContainer}>
             <div className={styles.innerContainer}>
                 <div className={styles.listContainer}>
-                    <List className={styles.list}>
+                    <List>
                         <ListItem
+                            key={0}
                             className={styles.listItem}
                             button
                             onClick={() => setSettings(METADATA)}
+                            selected={settings === METADATA}
                         >
                             Metadata
                         </ListItem>
                         <ListItem
+                            key={1}
```

Comment:
We don't need key props here. Those are only required when you have an array of elements.